---
title: "Final Report: Mirobiome of Red-Backed Salamanders"
author: "Allison Bogisich"
date: "November 27, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles. One page is about 500 words.

Bacterial communities which are host-associated have the potential to impact host health in many species. This is true across taxa including- but not limited to- humans [@fierer2012cross], corals [@rosenberg2007role], insects [@dillon2005diversity], and amphibians [@harris2009skin]. Skin-associated bacteria of amphibians are increasingly being recognized for their defensive role against pathogens, but we still have a very narrow understanding of their basic ecological functions. The cutaneous epidermal layer itself provides a barrier against pathogens, but it is the microbial communities that reside on the skin itself which provides a better defense. Particularly of concern to herpetologists is the pathogenic fungus *Batrachochytrium dendrobatidis* (Bd), which some symbiotic cutaneous-microbacteria appear to repell [@lam2010proportion,@woodhams2007symbiotic]. Bd is what causes the fungal disease chytridiomycosis, which has been a contributing factor in bringing many amphibian populations to near extinction within the past decade [@skerratt2007spread, @crawford2010epidemic].

Amphibians are well adapted to a semi-terrestrial life, because their skin is covered by mucous produced by numerous glands located over their entire body. These glandular secretions help protect the skin from mechanical trauma (such as abrasions), limits entry of pathogens via antimicrobial chemicals, and helps faciliate osmoregulation which occurs through the skin. Proper osmoregulation, which is centralized on an amphibians ventral 'drink patch', helps to maintain a proper salt and water balance within the internal organs when the amphibian is in water or on land [@dubaissi2011embryonic]. The antimicrobial compounds are components of an amphibian's innate immune response and defend amphibians against bacterial and fungal infections. Amphibians regularly undergo skin shedding which regulates populations of skin microorganisms and ensures the continued integrity of their skin surface [@cramp2014first]. Unlike other vertebrates, in which skin cells are cast off either individually or in small patches, amphibians undergo a more cyclical moulting process whereby an entire layer of skin is removed during every episode of shedding. Each time the skin microbiome population becomes reduced and returns to a lower level of diversity. Once separated, the outer stratum of the skin is lost to the environment, or in most cases is ingested by the organism in an attempt to reabsorb critical nutrients or parts of its own microbiome via digestion. Thus according to Cramp, risks of microbial imbalance post-molt on the skin are reduced. Since the vast majority of amphibians consume their own skin, it is possible that an integral cycle is established between the skin and the gut microbiome.

When operating optimally, the immune system-microbiota sybiosis combines the innate and adaptive branches of immunity in amphibians, enabling the system to better select for and calibrate adaptive immune responses and terminate maladaptive ones [@belkaid2014role]. However, once an amphibian becomes infected with Bd, the fungus spreads and grows over the drink patch, clogging the pores of skin through which osmorgeulation and respiration would normally take place. Consequentially, the organism effectively loses it's abiity to maintatin homeostasis and it's immune system becomes increasingly compromised, allowing Bd to garner more nutrients from the host for its own pathogenic life-cycle. Already commonly facing other threats like climate change and habitat encroachment, infection with Bd can be a critical hit to an already endangered species. However, some species of amphibians such as the red-backed salamander, *Plethodon cinereus*,  appear to have established symbiotic microbial communities which are providing another line of defense against infectious pathogens sharing a common host [@bataille2016microbiome]. While the microbiome of such species are resistant to pathogenic infection by Bd, it remains unclear as to why Bd can disrupt the microbiome of more susceptible species. Documented instances of such relationships during naturally occurring Bd outbreaks are few, and identifying causation from field observations is difficult. In this post-experimental work using Loudon et al.'s data from surveying the microbiome of *P. cinereus* [@loudon2014microbial], instead of reporting on the most prevalent and abundant OTUs, I will be investigating the relationship between the microbial makeup which was found on the salamanders and the microbial community in the medium upon which they were residing. I would hypothesize that the mircobial communities on the salamanders will have a markedly different diversity profile from that of the medium upon which they reside. Additionally, as the salamanders were tested over a period of time I will be examining the stability of their microbiomes over time. I'd predict that while the numerical quantities of the most present bacterial classes will change, that the relative amounts and ranking of classes will remain consistent over time as it would be adaptively advantageous to maintain the integrity of one's Bd resistant microbiome, as lapses would leave the organism more suceptible to infection.


# Methods

## Study design

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Sample origin and sequencing

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Computational

These are the methods you used to do your bioinformatic analyses analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

# Results

## Subsections are ok in the results section too

```{r your-code-here}
# Add code chunks as needed for your analyses
# For most analyses, I would recommend splitting the
# intensive computational part into a seperate R script file
# and then just load your libraries and the data object here
# using the `load()` function. Ask me for clarification if this is
# unclear.
```

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# See script 2 for more information about each package
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("citr")
library("dada2")
library("seqinr")
library("mctoolsr")
library("phyloseq")
# Be sure to load in the phyloseq data object created
load("output/phyloseq_obj.RData")
stopifnot(exists("phyloseq_obj"))
```

```{r make-alpha-diversity-plot}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
# alpha diversity metrics, create subset
salamander_subset_obj <- subset_samples(phyloseq_obj,
                                    Organism_s != "soil metagenome")
# plot alpha diversity of mucus samples
plot_richness(salamander_subset_obj,
                x = "env_biome_s",
                measures = c("Shannon", "Simpson"),
                color = "description_s") +
    xlab("Sample Origin") +
    geom_jitter(width = 0.2) +
    theme_bw()
```


  

In addition to a minimum of 5-10 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.



...frog skin contains natural substances that could be the basis for a powerful new genre of antibiotics.
...allies in humanity’s battle with antibiotic-resistant infections that threaten millions of people worldwide.
...been around 300 million years, so they’ve had plenty of time to learn how to defend themselves against disease-causing microbes in the environment. Their own environment includes polluted waterways where strong defenses against pathogens are a must


"One substance isolated from the skin secretions of the Foothill Yellow-legged Frog — a species once common in California and Oregon but now facing extinction — shows promise for killing methicillin-resistant Staphylococcus aureus (MRSA) bacteria. MRSA is a “superbug,” infamous for causing deadly outbreaks of infection among hospitalized patients. Now it is occurring in settings outside hospitals, including schools, nursing homes, and day care centers." [@conlon2014potential]

# Sources Cited
