---
title: "Final Report: Mirobiome Stability of Red-Backed Salamanders"
author: "Allison Bogisich"
date: "November 27, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles. One page is about 500 words.

Bacterial communities which are host-associated have the potential to impact host health in many species. This is true across taxa including- but not limited to- humans [@fierer2012cross], corals [@rosenberg2007role], insects [@dillon2005diversity], and amphibians [@harris2009skin]. Skin-associated bacteria of amphibians are increasingly being recognized for their defensive role against pathogens, but we still have a very narrow understanding of their basic ecological functions. The cutaneous epidermal layer itself provides a barrier against pathogens, but it is the microbial communities that reside on the skin itself which provides a better defense. Particularly of concern to herpetologists is the pathogenic fungus *Batrachochytrium dendrobatidis* (Bd), which some symbiotic cutaneous-microbacteria appear to repel [@lam2010proportion,@woodhams2007symbiotic]. Bd is what causes the fungal disease chytridiomycosis, which has been a contributing factor in bringing many amphibian populations to near extinction within the past decade [@skerratt2007spread, @crawford2010epidemic].

Amphibians are well adapted to a semi-terrestrial life, because their skin is covered by mucous produced by numerous glands located over their entire body. These glandular secretions help protect the skin from mechanical trauma (such as abrasions), limits entry of pathogens via antimicrobial chemicals, and helps facilitate osmoregulation which occurs through the skin. Proper osmoregulation, which is centralized on an amphibians ventral 'drink patch', helps to maintain a proper salt and water balance within the internal organs when the amphibian is in water or on land [@dubaissi2011embryonic]. The antimicrobial compounds are components of an amphibian's innate immune response and defend amphibians against bacterial and fungal infections. Amphibians regularly undergo skin shedding which regulates populations of skin microorganisms and ensures the continued integrity of their skin surface [@cramp2014first]. Unlike other vertebrates, in which skin cells are cast off either individually or in small patches, amphibians undergo a more cyclical molting process whereby an entire layer of skin is removed during every episode of shedding. Each time the skin microbiome population becomes reduced and returns to a lower level of diversity. Once separated, the outer stratum of the skin is lost to the environment, or in most cases is ingested by the organism in an attempt to reabsorb critical nutrients or parts of its own microbiome via digestion. Thus according to Cramp, risks of microbial imbalance post-molt on the skin are reduced. Since the vast majority of amphibians consume their own skin, it is possible that an integral cycle is established between the skin and the gut microbiome.

When operating optimally, the immune system-microbiota symbiosis combines the innate and adaptive branches of immunity in amphibians, enabling the system to better select for and calibrate adaptive immune responses and terminate maladaptive ones [@belkaid2014role]. However, once an amphibian becomes infected with Bd, the fungus spreads and grows over the drink patch, clogging the pores of skin through which osmoregulation and respiration would normally take place. Consequentially, the organism effectively loses it's ability to maintain homeostasis and it's immune system becomes increasingly compromised, allowing Bd to garner more nutrients from the host for its own pathogenic life-cycle. Already commonly facing other threats like climate change and habitat encroachment, infection with Bd can be a critical hit to an already endangered species. However, some species of amphibians such as the red-backed salamander, *Plethodon cinereus*,  appear to have established symbiotic microbial communities which are providing another line of defense against infectious pathogens sharing a common host [@bataille2016microbiome, @harris2009skin]. While the microbiome of such species are resistant to pathogenic infection by Bd, it remains unclear as to why Bd can disrupt the microbiome of more susceptible species. Documented instances of such relationships during naturally occurring Bd outbreaks are few, and identifying causation from field observations is difficult. In this post-experimental work using Loudon et al.'s data from surveying the microbiome of *P. cinereus* [@loudon2014microbial], instead of reporting on the most prevalent and abundant OTUs, I will be investigating the relationship between the microbial makeup which was found on the salamanders and the microbial community in the medium upon which they were residing. Developing more data in an effort to support the importance of environmental reservoirs for maintaining a hosts’ microbial community could have a direct effect on amphibian conservation research conducted in a laboratory setting, since critically endangered species are often taken into survival assurance colonies in captivity. Unfortunately, this removes them from their natural environment and places them in a setting that typically lacks similar bacterial reservoirs due to standard laboratory hygienics. This could greatly affect the structure of their microbiome, both in diversity and functionality. Currently, removal from their natural environment may likely be a major disturbance for their microbiota and may further limit the capacity for the microbiota to defend against pathogens. I would hypothesize that the microbial communities on the salamanders will have a markedly different diversity profile from that of the medium upon which they reside. Additionally, as the salamanders were tested over a period of time I will be examining the stability of their microbiomes over time. I'd predict that while the numerical quantities of the most present bacterial classes will change, that the relative amounts and ranking of classes will remain nearly constant over time as it would be adaptively advantageous to maintain the stability of one's Bd defensive microbiome, as lapses or large fluctuations would leave the organism more susceptible to infection. As some of an amphibians’ microbiota is capable of excreting anti fungal metabolites, the stability of the microbiota may be indicative of the health of the organism [@loudon2014microbial]. This analysis will report which of the microbiota were most stable despite changes in external(time) and internal (molting) conditions, as they may be the key to preventing more extinctions and reducing the spread of Bd.


# Methods

## Study design

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

*P. cinereus* is present in relative abundance in the Shenandoah mountain region of Virginia, and are in
close contact with soil and can tolerate even sterile laboratory conditions. Loudon et al. tested soil samples at salamander collection sites to determine whether a natural soil bacterial reservoir was necessary in order to maintain the stability and diversity of host bacterial communities. Previously, this had not been uniquely tested over time, by sampling salamanders in the field and again after captive-housing in a laboratory with or without being provided a bacterial reservoir. After field sampling, salamanders were housed either with soil from the field or with sterile media. Thereafter, they measured using both alpha and beta diversity using culture independent Illumina 16S rRNA gene sequencing to determine the diversity of bacterial organizational taxonomic units (OTUs) on each salamander and the relative microbial diversity between salamanders. Additionally, tests for the effectiveness of perturbations on shifting the stable core community were conducted on the collected specimens. Lastly, Loudon and his team tested whether the predicted anti-fungal functionality of the core community microbacteria was significantly distinct from the functions of the more peripheral microbial community members.


## Sample origin and sequencing

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

Salamanders were collected by Loudon and his team from the George Washington National Forest of the aforementioned mountain range in October of 2011 (VADGIF Permit No. 047519), and the soil in this experiment was sampled at the same location and time. After gloved capture in the field, the salamanders were rinsed three times with sterile media to remove and transient bacteria present on the specimen. In order to collect data on the immediate microbial environment, each salamander was quickly swabbed using ten strokes back and forth with a sterile rayon swab to collect data on the immediate microbial environment. They were swabbed on either their left or right ventral surface, side chosen at random. Swab samples were then stored on ice in the field and then frozen at negative eighty degrees Celsius prior to DNA extraction. The collected salamanders were placed into sterile 50-mL falcon tubes, and the soil samples were transported in plastic autoclaved containers. The salamanders were then housed individually and randomly assigned to one of two experimental conditions designed to test whether the bacterial reservoir affects the composition and stability of the specimen's microbiome. Ten were assigned to live on sterile media in their containers, while the other ten in the soil treatment group lived on homogenized soil from the salamander's natural habitat. Initial soil samples were taken in triplicate prior to being deposited into the containers in order to identify bacterial communities. The salamanders were swabbed every seven days for 28 days, and 'sterile media' was changed every seven days post-swabbing. Soil media was not replaced. Soil was sampled in triplicate every fourteen days for the soil treatment group. Every salamander was fed 15 fruit flies each week post-sampling, although the bacteria from the flies did not reportedly contribute significantly to the microbial community given the bacterial biomass in the soil.

DNA extractions and 16S rRNA amplifications were performed with the addition of primers, and sequenced on an Illumina HiSeq 2000. One aliquot of DNA was also used to test each salamander for the presence of Bd. These data can be found in the Quantitative Insights Into Microbial Ecology (QIIME) database (www.microbio.me/qiime; study no. 1618) or in the European Bioinformatics Institute (EBI) archive with the accession number ERP003771. Amplicons were sequenced using a third of an Illumina HiSeq lane at the University of Colorado at Boulder, yielding short 100-bp reads. Quantitative Insights Into Microbial Ecology (QIIME) version 1.5.0 [@caporaso2010qiime] was used for all subsequent sequence analysis, unless otherwise noted by Loudon et al.. Sequences were filtered for quality and assigned to their respective sample using the default settings of QIME. 


## Computational

These are the methods you used to do your bioinformatic analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

Post DNA extraction from the samples, sequences were downloaded via a bash script for-loop from NCBI in order to acquire the sequenced samples within a pre-formatted SRA run table. The sequences were then trimmed and run through a DADA2 pipeline using an R script. Primary packages used to clean and analyze the data were installed via bio conductor and devtools. The DADA2 package [@callahan2016dada2] was installed for use in conducting fast and accurate sample inferences from amplicon data with single-nucleotide resolution, and to `ploterrors` and `plotQualityProfile`. Seqinr was used to export fastas of our final de-noised variants. Mctoolsr, which stands for “microbial community analysis tools in R”, was used to more efficiently manipulate data and perform certain analyses common to microbial community studies, while allowing us to easily access data using established functions (https://github.com/leffj/mctoolsr/). The 'ggplot2' package could then be used for visualization of data into graphic results [@wickham2016ggplot2]. Phyloseq [@mcmurdie2013phyloseq] was the final package installed for my pipeline in order to visualize my results using tools `plot_bar`, `plot_richness`.
 
Raw data were sorted for order and samples were extracted with their appropriate naming system. Quality checks were run prior to any data visualization to ensure that the lengths of the sequences being used were of proper length for obtaining quality scores above 30, which required filtering and trimming all sequences. Error models were also created to analyze each sample individually, and then visualized by plotting. Duplicated sequences and chimeras were then removed from the data set, so as not to compound error. Sequences were removed if shorter than 55 bp or larger than 155 bp in length, had a quality score lower than 25, had ambiguous characters or uncorrectable barcodes, or if it did not contain the primer sequence. Only one sample- ERR325545- from the Loudon et al. study was removed due to quality issues.

# Results

## Subsections are ok in the results section too
In addition to a minimum of 5-10 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

```{r your-code-here}
# Add code chunks as needed for your analyses
# For most analyses, I would recommend splitting the
# intensive computational part into a seperate R script file
# and then just load your libraries and the data object here
# using the `load()` function. Ask me for clarification if this is
# unclear.
```

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# See script 2 for more information about each package
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("citr")
library("dada2")
library("seqinr")
library("mctoolsr")
library("phyloseq")
# Be sure to load in the phyloseq data object created
load("output/phyloseq_obj.RData")
stopifnot(exists("phyloseq_obj"))
```

```{r make-salamander-alpha-diversity-plot}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
# alpha diversity metrics, create subset
salamander_subset_obj <- subset_samples(phyloseq_obj,
                                    Organism_s != "soil metagenome")
# plot alpha diversity of mucus samples
plot_richness(salamander_subset_obj,
                x = "env_biome_s",
                measures = c("Shannon", "Simpson"),
                color = "description_s") +
    xlab("Sample Origin") +
    geom_jitter(width = 0.2) +
    theme_bw() + 
    ggtitle(" Salamander Micriobiota Alpha Diversity Across Substrates") +
    theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1))
```
**Figure 1**: Alpha diversity measure of the bacterial community diversity on individual salamanders.

By using the Shannon and Simpson diversity indexes, I have plotted the mean microbial species diversity in samples at the level of an individual salamander. This also includes sampling from a control environment in which we'd expect to see no diversity as well as water samples that were collected from salamanders which were taken from an aquatic habitat in the field. The green data points indicate samples from salamanders in the sterile media treatement group that were without a bacterial reservoir. The teal point are from samples taken from salamanders which were provided the soil and it's associated bacterial reservoir from their natural habitat. Some of the control samples appear to have a high amount of diversity while others have none in the Shannon index, but all display alpha diversity in the Simpson index. The alpha diversity distribution among salamander samples appears to be fairly heterogeneous. In both treatment groups there were samples that had low and high diversity measures across both indexes. For the aqautic biome sampling, the alpha diversity remained relatively high across both indexes. 


```{r make-substrate-alpha-diversity-plot}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
# alpha diversity metrics, create subset
soil_subset_obj <- subset_samples(phyloseq_obj,
                                    Organism_s != "organismal metagenomes")
# plot alpha diversity of soil samples
plot_richness(soil_subset_obj,
                x = "env_biome_s",
                measures = c("Shannon", "Simpson"),
                color = "description_s") +
    xlab("Sample Origin") +
    geom_jitter(width = 0.2) +
    theme_bw() + 
    theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1)) +
    ggtitle("Substrate Alpha Diversity Across Conditions")
```  
**Figure 2**: Alpha diversity measure of the bacterial community diversity across substrate media.

The breakdown of the alpha diversity of substrate sampling looks a little bit different from that of the salamander alpha diversity. Similarly we can see that the controls for substrate also had contaminants and exhibited a measure of diversity. Lab soils were broken down into two separate descriptions due to an typo during the cataloging of the data with the lack of capitalization of the "s" in "soil". However, we can still see that the alpha diversity of the lab soils from the treatment group with the bacterial reservoir did indeed have higher alpha diversity than that of the non-reservoir treatment group of salamanders in media. Both the field and lab soils had almost six times as much bacterial alpha diversity relative to the sterile media in the Shannon index, and 0.5 times as much according to the Simpson index. The water tested from the aquatic biome fell in the middle of distribution for alpha diversity in the Shannon index, with greater diversity than the sterile media but less than the field lab tested soils. In the Simpson index, the difference in alpha diversity between water and soils is less pronounced.


```{r top-15-core-community-bacteria}
# here I want to pull out just the top 15 types
# of bacterium that were persistant
# have to filter it out as a subset from final_table
# using psmelt to combine taxa with abundance levels
melted_obj <- psmelt(phyloseq_obj)

# once object is created I can create top 15 list
top_bacteria <- melted_obj %>%
 group_by(Phylum, Class) %>%
 summarize(sum_count = sum(Abundance)) %>%
 arrange(desc(sum_count)) %>%
 head(15)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
top_bacteria %>%
 kable()

```
**Table 1**: The top 15 most abundant bacteria across all salamander and substrate samples.

While the most abundant sequence identified did not have a matching known bacterial sequence, the rest of the top fourteen display the most common bacterial phylum and associated classes. These were the most prolific taxa across both sampled skin swabs from salamanders and also from sampled substrates. The most common phyla were Proteobacteria which took five of the fifteen spots for greatest abundance. Actino- and Acidobacterium were also well represented with 1:3 respectively. Bacteriodetes and Verrucomicrobia were the remaining phyla represented. The most common classes for the respective phyla did not show much of a trend, although there were three taxonomic groupings of the class Acidobacteria which were prominent.


```{r barplot-of-salamander-bacterial-profile, echo = FALSE}
# create subset with only salamander bacteria
salamander_bacteria_obj <- melted_obj %>%
  filter(grepl("organismal metagenomes", Organism_s)) %>%
  group_by(Phylum, Class, Order, Genus, treatment_description_s) %>%
  summarize(sum_count = sum(Abundance)) %>%
  arrange(desc(sum_count)) %>%
  head(n = 50)


# this code uses the same data as above, but use it to make a
# barplot
salamander_bacteria_obj %>%
  ggplot(aes(x = treatment_description_s,
             y = sum_count,
             fill = Class)) +
  geom_col(position = "dodge") +
  theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust = 1))
  ggtitle(" Microbe Class Diversity Across Time in Salamanders")

```
**Figure 3**: Presence of microbial classes on salamanders over time in both experimental conditions.

In order to compare the microbial communites found on the salamanders over a period of four weeks, I broke it down by representative classes. For the salamanders in the media treatment, the majority of the classes of bacteria present during the intial collection were still present on the sampled salamanders even almost month later in sterile codtions. Gammaproteobacteria, Sphingobacteriia, and Actinobacteria were the core community. The greatest variety of bacterial classes were found on the intial day of sampling. In the soil treated salamanders, there also appears to be the most class diversity on the day of collection. However, by day 28 only one sequence matched type of bacteria remained- Acidobacteria_Grp2. The sum count for the bacteria taxa were quite small overall for both experimental groups of salamanders.

```{r barplot-of-soil-bacterial-profile, echo = FALSE}
# create subset with only salamander bacteria
soil_bacteria_obj <- melted_obj %>%
  filter(grepl("soil metagenome", Organism_s)) %>%
  group_by(Phylum, Class, Order, Genus, treatment_description_s) %>%
  summarize(sum_count = sum(Abundance)) %>%
  arrange(desc(sum_count)) %>%
  head(n = 50)


# this code uses the same data as above, but uses it to 
# make a barplot
soil_bacteria_obj %>%
  ggplot(aes(x = treatment_description_s,
             y = sum_count,
             fill = Class)) +
  geom_col(position = "dodge") +
  theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust = 1))
  ggtitle(" Microbe Class Diversity Across Time in Soil")

```
**Figure 4**: Presence of microbial classes over time in soil.

When analyzing the relative amounts of bacterial classes present in the soil tested over time, there are much larger summation values for the classes. Counts ranged anywhere from 13,000 at the lowest to nearly a million at the highest for soil samples. While the field soil distribution had quite a large diversity of microbial classes (seven identified), samples from day zero in the lab dropped down to just one class- Acidobactera_Grp2. However, then when sampled in the subsequent weeks the soil regained a great deal of microbial diversity which persisted till the end of the study. Weeks two and four had samples that both exhibited essentially equivalent amounts of the ten identified classes present. Acidobactera_Grp2 and Grp1 consistently exhbited the greatest abundance in the sampled soils once in the laboratory setting.

# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

Host microbiota performs a number of important functions for their hosts, such as disease resistance
(Dillon et al., 2005; Rosenberg et al., 2007; Harris et al., 2009a,b; Becker and Harris, 2010),
metabolism, vitamin production, development and activity of the immune system and behavior (Turnbaugh et al., 2007). In this study, I examined the bacterial community dynamics on healthy, red-backed salamanders that were not infected with Bd. It is likely that the stability and diversity of host microbiota are related to the consistency and quality of protection. In both the soil and the sterile media treatments, the community composition of the salamander skin microbiota changed when brought into the
laboratory. Thereafter, alpha diversity remained stable over the course of the experiment
for salamanders maintained in the presence of a soil bacterial reservoir. These results suggest that
relatively minor disturbances, such as skin sloughing (Meyer et al., 2012), which occurred during the
experiment in the laboratory, do not affect diversity if a bacterial reservoir is available. In the field, there was no correlation between alpha diversity of salamanders’ skin microbiota and
the microbiota of the salamanders’ immediate habitat upon capture in the field.
Thus, even though their microbes are environmentally derived, salamander skin communities are
not determined solely by passive inoculation. Rather, host factors appear to be selected for and
maintain host-associated microbes at similar relative abundance over time. The same pattern has been
found in humpback whales and has been suggested to occur in amphibian larvae (Apprill et al., 2011).

The results support my hypothesis that the core microbiota of the salamanders would remain relatively stable over time. Gammaproteobacteria, Sphingobacteriia, and Actinobacteria were the core community (Figure 3).

I'd predict that the diversity of microbial classes present on sterile media salamanders remained higher that on those of the soil treated salamanders because....

...frog skin contains natural substances that could be the basis for a powerful new genre of antibiotics.
...allies in humanity’s battle with antibiotic-resistant infections that threaten millions of people worldwide.
...been around 300 million years, so they’ve had plenty of time to learn how to defend themselves against disease-causing microbes in the environment. Their own environment includes polluted waterways where strong defenses against pathogens are a must [@colombo2015microbiota].

.....n all-D-enantiomer is nearly inactive against Gram-negative bacteria and some Gram-positive strains but is fully active against fungi and other Gram-positive bacteria, suggesting that more than one mechanism accounts for the antimicrobial activity of this peptide [@fehlbaum1996structure].

One substance that has been isolated from the skin secretions of the Foothill Yellow-legged Frog — a species once common throughout the mountainous regions in California and Oregon, but which are now facing extinction — shows promise for killing antibiotic-resistant *Staphylococcus aureus* (MRSA) bacteria. MRSA is a more commonly called a “superbug,” and has become infamous for causing deadly outbreaks of infection among already hospitalized patients. Now that outbreaks have begun occurring in settings outside of hospitals, including schools, nursing homes, and day care centers, more concern has been mounting about how to best combat the pathogen [@conlon2014potential]. 


# Sources Cited
